README.md

```diff
@@ -26,6 +26,11 @@ Deploy this bundle using juju-quickstart:
 
     juju quickstart realtime-syslog-analytics
 
+Alternatively, you can deploy using **conjure-up**
+
+    apt install conjure-up
+    conjure-up analytics
```

Comment:
do you really want just `analytics` here?  don't get me wrong, i love it, but it's a pretty broad term... perhaps `syslog-analytics` or even `bigdata-syslog-analytics` would be more appropriate.